examples/opensora_cai/opensora/models/stdit.py

```diff
@@ -284,7 +284,7 @@ def __init__(self, in_channels, hidden_size, uncond_prob, act_layer=nn.GELU(appr
 
         y_embedding = ops.randn(token_num, in_channels) / in_channels**0.5
         # just for token dropping replacement, not learnable
-        self.y_embedding = ms.Tensor(y_embedding, dtype=ms.float32)
+        self.y_embedding = ms.Parameter(ms.Tensor(y_embedding, dtype=ms.float32), requires_grad=False)
```

Comment:
this y_embedding should be trainable.

Reply:
no, y_embedding  is wrapped with register_buffer params in torach, which is not trainable by default